src/balmy/utils.ts

```diff
@@ -40,3 +41,13 @@ export function isCompleteToken(token: TokenData) {
     token.symbol
   );
 }
+
+export function sortTokens<T extends { chainId: ChainId; address: string }>(tokens: T[]): T[] {
+  return tokens.sort((a, b) => {
+    if (a.chainId !== b.chainId) {
+      return a.chainId - b.chainId;
+    } else {
+      return a.address.toLowerCase().localeCompare(b.address.toLowerCase());
+    }
+  });
```

Comment:
```suggestion
  return tokens.sort((a, b) => (a.chainId - b.chainId) || a.address.toLowerCase().localeCompare(b.address.toLowerCase()))
  });
```
Small suggestion, but ignorable